ahcore/cli/tiling.py

```diff
@@ -363,7 +365,7 @@ def _tiling_pipeline(
             tile_size=dataset_cfg.tile_size,
             tile_overlap=dataset_cfg.tile_overlap,
             num_samples=len(dataset),
-            is_compressed_image=compression != "none",
+            data_format=DataFormat.COMPRESSED_IMAGE if compression != "none" else DataFormat.IMAGE,
             color_profile=color_profile,
```

Comment:
Doesn't 'none' need to be an Enum?

Reply:
Fixed, I've looked at it and here the dataformat should always be Image, because the compression here does not make the image a compressed image.